src/tls/acme/client.rs

```diff
@@ -333,19 +333,21 @@ impl Client {
         names: &Vec<String>,
         private_key: Option<Vec<u8>>,
     ) -> Result<Cert, Error> {
-        // Perform the issuance
+        // Pre-cleanup: attempt to remove all existing TXT _acme-challenge records for the given names.
```

Comment:
Let's fix the comment also to remove any mention of DNS TXT stuff. We operate just on abstract tokens.